<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameOver</class>
 <widget class="QDialog" name="GameOver">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>500</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>410</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="exit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>430</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QLabel" name="score">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>371</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="newGame">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>430</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>141</width>
     <height>41</height>
    </size>
   </property>
   <property name="text">
    <string>New Game</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="highScores">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>371</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameInput">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>140</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="subScore">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>140</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
